### ISSUE TITLE
Search problems

### ISSUE THREAD

**Issue by ligi** [ethb3rlin/Submi-ions#14]
![image](https://github.com/ethb3rlin/Submi-ions/assets/111600/d8ffae9b-abc7-44c4-9242-f3c8ecad7fed)
![image](https://github.com/ethb3rlin/Submi-ions/assets/111600/ebb804ee-9220-4ca8-9c9c-f3784d0bef27)

**Comment by q9f** [comment]
yeah, I also noticed this. sometimes you had to type in some substrings of the term you search to actually find the project.
